react/Minicart.tsx

```diff
@@ -106,6 +107,17 @@ const Minicart: FC<MinicartProps> = ({
             >
               {children}
             </DrawerMode>
+          ) : variation === 'popupWithLink' ? (
```

Comment:
My suggestion is to do something like that on the current PopupMode
```suggestion
          ...(variation === 'popupWithLink' ? { variation } : {})
```

Reply:
Also lint this file

Reply:
Hello Igor !

I don't understand your suggestion, could you give another example, please?

Reply:
Sorry, it was a mistake, I thought that the Popup component was duplicated, forgive me.